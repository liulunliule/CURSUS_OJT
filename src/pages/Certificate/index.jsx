import certificate from "../../assets/img/Certificate/certificate.jpg";
import React from "react";
import "./index.scss";

const Certificate = () => {
  return (
    <div className="certificate">
      <img src={certificate} />
    </div>
  );
};

export default Certificate;
