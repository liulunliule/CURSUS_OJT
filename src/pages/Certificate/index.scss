.certificate {
    margin: 0px;
    background-color: rgb(14, 14, 14);
    img{
        height: 100vh;
        display: block;
        margin: auto;
        cursor: zoom-in;
        background-color: hsl(0, 0%, 90%);
        transition: background-color 300ms;
    }
}