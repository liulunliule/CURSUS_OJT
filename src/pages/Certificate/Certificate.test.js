import React from 'react';
import { render } from '@testing-library/react';
import Certificate from '.';

test('renders Certificate component correctly', () => {
  const { asFragment } = render(<Certificate />);
  expect(asFragment()).toMatchSnapshot();
});
