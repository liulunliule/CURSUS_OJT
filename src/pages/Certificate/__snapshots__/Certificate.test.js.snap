// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Certificate component correctly 1`] = `
<DocumentFragment>
  <div
    class="certificate"
  >
    <img
      src="certificate.jpg"
    />
  </div>
</DocumentFragment>
`;
